cephci/prereq.py

```diff
@@ -191,7 +191,7 @@ def setup_private_container_registry(
         private_reg_username (str): private registry username
         private_reg_password (str): private registry password
         docker_reg_image (str): docker-registry image name
-        images (str): list of images
+        images (list): list of images
```

Comment:
doesn't it affect existing tests ?

Reply:
Its a typo fix Vaibhav, it was already accepting list of images. The description explains the same